packages/orbit-tailwind-preset/README.md

```diff
@@ -0,0 +1,52 @@
+# Orbit tailwind preset
+
+This package contains the Orbit Tailwind presets.
+
+Currently, there are two presets available:
+
+- `orbitPreset` - Orbit's foundation styles, including typography, colors, spacing, etc.
+- `orbitComponentsPreset` - Orbit's component tokens. This preset is mainly for Orbit's internal usage and
+  migration to Tailwind. It already includes the `orbitPreset` and adds component-level tokens.
+
+## Installation
+
+```bash
+yarn add -D @kiwicom/orbit-tailwind-preset
+```
+
+or with npm:
+
+```bash
+npm install --save-dev @kiwicom/orbit-tailwind-preset
+```
+
+## Usage
+
+In your `tailwind.config.js` file, add the following:
+
+```js
+const orbitPreset = require("@kiwicom/orbit-tailwind-preset");
+
+module.exports = {
+  presets: [orbitPreset],
+};
+```
+
+We also have preset with component tokens, which means to be used inside Orbit components. In case you need to use our component-level tokens, add the `orbitComponentsPreset` to your `tailwind.config.js` file:
+
+```js
+const { orbitComponentsPreset } = require("@kiwicom/orbit-tailwind-preset");
+
+module.exports = {
+  presets: [
+    orbitComponentsPreset({
+      // Enable normalizing of the browser's default styles, which is disabled by default
+      disabledPreflight: false,
+    }),
+  ],
+};
+```
+
```

Comment:
Wasn't there more documentation text here? I feel this is missing a connection between this and the snippet below... I feel it only tells how to import the preset but then not how to use it. It just has a snippet that seems a bit out of context and short. WDYT?

Reply:
Im planning to do that in separate MR :) adding more docs